Add advisory regarding OVH Game DDoS incompatibility (#273)
GeyserMC/GeyserWiki: _docs/geyser/port-forwarding.md

OVH:
1. Navigate to `Network interfaces`
2. Click on the `...` button on the table for your IP -> then `...` and `Configure the GAME firewall` -> `Add rule` -> `Other protocol` ~~(or `minecraftPocketEdition` if available)~~
3. Add your Geyser port into `outgoing port`.

SoYouStart (subsidiary of OVH):
1. Click the IP tab.
2. Click the gear at the right of the public IP address; select "Game mitigation".
3. Pick "Add a rule".
4. Select "minecraftPocketEdition" in the dropdown list and enter the target UDP ports.
5. Save and wait a few seconds for the changes to come into effect.

#### OVH/SoYouStart Filter Incompatibility Issue

> The `minecraftPocketEdition` rule/filter/game type currently does not work and you must use the `Other` type.  

Reviewer: This sounds a bit unclear - since this only applies to the "GAME" firewall

> If you would like to continue using `minecraftPocketEdition`, you may disable the incompatible security feature by adding `-DGeyser.RakSendCookie=false` to your startup flags for Geyser (works regardless of whether you use plugin or standalone Geyser).

Reviewer: ```suggestion
If you would like to continue using the filter type `minecraftPocketEdition`, you may disable the incompatible security feature by adding `-DGeyser.RakSendCookie=false` to your Java server's (or Geyser Standalone proxy's) startup flags.
```

For more information see:  

>     [This issue on OVH's infrastructure roadmap](https://github.com/ovh/infrastructure-roadmap/issues/186)  
>     [The pull request which implemented the security feature that caused the incompatibility](https://github.com/GeyserMC/Geyser/pull/4554)

Reviewer 2: ```suggestion
 - [This issue on OVH's infrastructure roadmap](https://github.com/ovh/infrastructure-roadmap/issues/186)  
 - [The pull request which implemented the security feature that caused the incompatibility](https://github.com/GeyserMC/Geyser/pull/4554)
```
